src/Microsoft.XmlSerializer.Generator/src/Sgen.cs

```diff
@@ -137,6 +160,7 @@ private void GenerateFile(List<string> typeNames, string assemblyName, bool prox
                 try
                 {
                     types = assembly.GetTypes();
+                    Console.Out.WriteLine(string.Format("{0} types loaded", types.Count()));
```

Comment:
I don't see why this information would be helpful. Why do we want to output this line?